packages/user-interface/src/components/Notification.tsx

```diff
@@ -0,0 +1,44 @@
+import Alert, { AlertProps } from "@gemeente-denhaag/alert";
+import styles from "./Notification.module.scss";
+import { useEffect, useState } from "react";
+import classNames from "classnames";
+
+interface Props extends AlertProps {
+  closable?: boolean;
+}
+
+const Notification = ({
+  closable = true,
+  title,
+  text,
+  variant,
+  className,
+  ...props
+}: Props) => {
+  const [hideNotification, setHideNotification] = useState(false);
+  const alertClassName = classNames(
+    styles["nl-portal-notification"],
+    className,
+  );
+
+  useEffect(() => {
+    scrollTo(0, 0);
```

Comment:
Scroll alleen afvuren wanneer showNotification true is? Anders gaat hij ook scrollen als je hem dicht doet.

Reply:
o ja :)

Reply:
O ja :)